blog.html

```diff
@@ -66,6 +66,8 @@
                         </p>
                         <section class="light">
                             <ul style="list-style-type: none;">
+                                <li class="leading-block"><span class="blog-post-date">7 July, 2021</span> <a href="posts/qutip.html">QuTiP's 10-year anniversary</a>
```

Comment:
```suggestion
                                <li class="leading-block"><span class="blog-post-date">7 July, 2021</span> <a href="posts/qutip-10th-anniversary.html">QuTiP's 10-year anniversary</a>
```
Could we use a less generic filename for the blog post? It might help in future if we ever want to post something about QuTiP again? ;)

Reply:
Added it, thanks.